lib/components/home/artwork_rails/artwork_rail.js

```diff
@@ -54,13 +54,13 @@ class ArtworkRail extends React.Component {
     let url = null
     switch (key) {
       case 'followed_artists':
-        url = '/me/notifications'
+        url = 'works-for-you'
```

Comment:
I should think about a better way to do this so these two files can share logic.